libraries/session-storage/impl/src/main/sqldelight/io/element/android/libraries/matrix/session/SessionData.sq

```diff
@@ -1,13 +1,20 @@
+------------------------------------
```

Comment:
I understand the desire of keeping track of things, but this comment will only add additional burden to any future sqldelight migration (you may also forget to update the comment and things will then appear out of sync).

The current version of the DB is always represented by the last *.db file.
Every time you commit a change to the .sq files you must always create a new x.sqm and then generate x+1.db and commit those as well.

Reply:
+1

Reply:
You're right. I have updated the comment, also to make the process more visible (it's in the gradle file now)

12996e825353c71f1a5f59d650d500721fcb9849